examples/bin/readpw.sh

```diff
@@ -0,0 +1,3 @@
+# shellcheck shell=bash disable=SC2034
```

Comment:
```suggestion
#!/usr/bin/env bash

# shellcheck shell=bash disable=SC2034
```

Reply:
I did not put a shell on it because its not really executable. It can only be sourced. If you execute it, it runs, but the env var gets undefined at the end.